components/dashboard/src/admin/BlockedRepositories.tsx

```diff
@@ -148,8 +148,10 @@ export function BlockedRepositoriesList(props: Props) {
                 </div>
             </div>
 
-            <Alert type={"info"} closable={false} showIcon={true} className="flex rounded p-2 w-2/3 mb-2 w-full">
-                <span>Search entries by their Repositoriy URL regular expression (RegEx).</span>
+            <Alert type={"info"} closable={false} showIcon={true} className="flex rounded p-2 mb-2 w-full">
+                <span>
+                    Search entries by their repository URL <abbr title="regular expression">RegEx</abbr>.
```

Comment:
Here besides fixing a typo, an abbreviation was added so that we both save space and utilize semantic HTML for abbreviations.

Reply:
**thought**: Not particularly proud of how this is rendering by default (see screenshots below) or using this controversial abbreviation design pattern[[1](https://microformats.org/wiki/abbr-design-pattern)] as it comes with some accessibility concerns, but let's keep it for now until we have a better way to present this information. 🤝 

| Chrome | Firefox | Safari |
|-|-|-|
| <img width="400" alt="a" src="https://user-images.githubusercontent.com/120486/186999590-c80a9f31-5c33-4e5b-bf5e-9401fcb4bdf7.png"> | <img width="400" alt="b" src="https://user-images.githubusercontent.com/120486/186999596-e3113c2f-cb79-4f8e-8457-a2ccf472cddc.png"> | <img width="400" alt="c" src="https://user-images.githubusercontent.com/120486/186999602-106f2e56-365a-4d21-bbfc-4ccdd43d7c6d.png"> |

Reply:
Thanks for introducing me to Microformats! It's a really great guide from what I'm seeing.